app/views/layouts/_adv_search_footer.html.haml

```diff
@@ -6,18 +6,21 @@
 #adv_search_footer
   - if mode == "search"
     - if @edit[@expkey].available_adv_searches.blank? && report_expressions.blank?
-      = button_tag(_("Load..."),
+      = button_tag(_("Load"),
                    :class => "btn btn-primary disabled pull-left",
                    :alt   => t = _("No saved filters or report filters are available to load"),
                    :title => t)
     - else
-      = button_tag(_('Load...'),
+      = button_tag(_('Load'),
                    :class   => "btn btn-primary pull-left",
                    :alt     => t = _("Load a filter"),
                    :title   => t,
                    :onclick => "miqAjaxButton('#{url_for(:action => 'adv_search_button', :button => "load")}');")
     - if @edit[@expkey][:exp_table].flatten.first == "???"
-      = button_tag(_("Apply"), :class => "btn btn-primary disabled")
+      = button_tag(_("Apply"),
```

Comment:
@mshriver Found the button at last, hope it's the right one 😄 

![screencast from 2017-02-13 18-38-59](https://cloud.githubusercontent.com/assets/1187051/22895342/6ac2f78e-f21c-11e6-8bc7-523668bbf603.gif)